src/test/kotlin/cacophony/semantic/FunctionAnalysisTest.kt

```diff
@@ -30,7 +30,7 @@ class FunctionAnalysisTest {
             )
     }
 
-    @Test
```

Comment:
Why are these lines commented out?

Reply:
These tests don't make any sense now as function don't keep info about usage of variables created by itself.
(for now every test return just returns function with empty variables set)
However, when the whole AST will be the function, these tests will be adjusted appropriately, so I thought there is no need to remove them completely just to write almost identical tests in a moment.